### PR TITLE
Feature/apple login

### PR DESCRIPTION
**Pull request by SukitOwl**

### CODE REVIEW

File: Cpp/include/odin/apple.hpp [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/**
+    Copyright 2018-2020 Red Anchor Trading Co. Ltd.
```
Comment (KayEss) [inline Cpp/include/odin/apple.hpp:2]: Should be just 2020

File: Cpp/odin-fg/native.expect.cpp [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+#include <odin/fg/native.hpp>
+
+#include <fost/insert>
+#include <fost/ua/cache>
+#include <fost/ua/cache.detail.hpp>
+
+
+const fg::frame::builtin odin::lib::expect = [](fg::frame &stack,
```
Comment (KayEss) [inline Cpp/odin-fg/native.expect.cpp:16]: Why do we need this here? Shouldn't this be in the fostgres repo?

File: Cpp/odin-views/app.facebook.cpp [hunk 3]
```diff
@@ -1,6 +1,5 @@
 /**
     Copyright 2019-2020 Red Anchor Trading Co. Ltd.
-
```
Comment (KayEss) [inline Cpp/odin-views/app.facebook.cpp:3]: Don't remove this blank line

File: Cpp/odin-fg/native.expect.cpp (hunk 2 above)
Comment (SukitOwl) [reply Cpp/odin-fg/native.expect.cpp:16]: I will check

**Pull request merged**